TulparAudio/source/TulparConfigurator.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+* Copyright (C) 2018 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#include <tulpar/TulparConfigurator.hpp>
+
+#include <tulpar/InternalLoggers.hpp>
+
+#include <AL/alc.h>
+
+namespace tulpar
+{
+
+TulparConfigurator::TulparConfigurator()
+    : bufferBatch(32)
+    , sourceBatch(32)
+    , device(TulparConfigurator::Device
+        {
+            std::string()
+            , true
```

Comment:
I'd rather see enum here, braced initialization looks a lot less self-descriptive than it could have been.

Reply:
It's ambiguous for a reader that is not familiar with the class, what this Boolean parameter means. I don't think that's waiting for c++20 feature is a solution for this either. I'd rather see an enum class that is derived from bool, just to be able to say what this means on initialization, or static value for that matter. I don't like seeing pure rvalues in the initialization code, unless it's for performance reasons, which it is not, cause it's bool.

Reply:
Will add constructor and `static constexpr bool` aliases for values.